### PR TITLE
Improvements to liveliness tests [6275]

### PR DESCRIPTION
**Pull request by raquelalvarezbanos**

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/345/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/318/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/343/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/349/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/322/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/347/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/350/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/323/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/348/)

File: test/blackbox/BlackboxTestsLivelinessQos.cpp [hunk 1]
```diff
@@ -30,18 +30,17 @@ TEST(LivelinessQos, Liveliness_Automatic_Reliable)
     PubSubWriter<HelloWorldType> writer(TEST_TOPIC_NAME);
 
     // Liveliness lease duration and announcement period
-    uint32_t liveliness_ms = 200;
-    Duration_t liveliness_s(liveliness_ms * 1e-3);
-    Duration_t announcement_period(liveliness_ms * 1e-3 * 0.01);
+    unsigned int lease_duration_ms = 1000;
```
Comment (LuisGP) [inline test/blackbox/BlackboxTestsLivelinessQos.cpp]: I would prefer to use `uint32_t` instead of `unsigned int`. But it is up to you.

Comment (raquelalvarezbanos) [reply test/blackbox/BlackboxTestsLivelinessQos.cpp]: OK, I'll use `uint32_t` in the future

**Pull request merged**